tests/containers/bci_test.pm

```diff
@@ -10,92 +10,56 @@
 #   the lock-in imposed by alternative offerings.
 #
 #   This module is used to test BCI repository and BCI container images.
-#   It makes the call to tox to run the different test environments and
-#   parses the resulting JUnit logs at the end.
+#   It makes the call to tox to run a specific test environment.
 # Maintainer: qa-c team <qa-c@suse.de>
 
 use Mojo::Base qw(consoletest);
-use XML::LibXML;
+use Time::HiRes qw(clock_gettime CLOCK_MONOTONIC);
```

Comment:
Not sure why we use `Time::HiRes`, we only display seconds unit in the end.
a hint:
`use Time::HiRes qw(time)` overwrite your current `time()` function which return then a float like `<sec>.<ms>`